### ISSUE TITLE
Interpolation tutorial omits important info on lerp()

### ISSUE THREAD

**Issue by verity147** [godotengine/godot-docs#10362]
**Godot 4.3:**

**Issue description:**
The description and example for Vector2.lerp() makes it seem like the function automatically stops once the weight parameter hits or exceeds 1
or the target parameter is reached, mostly due to the looping gif and the minimalist code example.

I'd suggest at least mentioning that the user needs to actively limit their weight property or otherwise stop the lerp manually.

Suggested changes to the code example:
```
var t = 0.0

func _physics_process(delta):
	clamp(t += delta * 0.4, 0, 1)

	$Sprite2D.position = $A.position.lerp($B.position, t)
```
or checking if the target position is reached etc. But I feel like clamping t is the most universally applicable way to solve this.

I feel like a lot of the issue stems from the gif with the line "It will produce the following motion:" above it suggesting that there is some kind of automatic stop to the lerp.

This has been mentioned before in [#9674](https://github.com/godotengine/godot-docs/issues/9674) but the person reporting closed it themselves on the same day.

**URL to the documentation page (if already existing):
https://docs.godotengine.org/en/stable/tutorials/math/interpolation.html**

**Comment by AThousandShips** [comment]
The [method documentation](https://docs.godotengine.org/en/latest/classes/class_vector2.html#class-vector2-method-lerp) says to limit the weight to the `[0, 1]` range

**Comment by tetrapod00** [comment]
Yeah, it seems like the class reference and the tutorial could be clearer in this case. The documentation for the global scope [lerp()
](https://docs.godotengine.org/en/latest/classes/class_%40globalscope.html#class-globalscope-method-lerp) is already quite good IMO. So maybe we should copy elements of that, or link to it, for the Vector2.lerp() description (and the similar ones for the other vector lerps).

The tutorial page has needed a rewrite for a while anyway.

As for adding a warning, or clamping, or similar: it is not desirable for a general purpose `lerp()` function to clamp, since lerp() is a mathematical function and using values outside the range is valid. The docs for global scope lerp even mention that.
